crates/ide/src/syntax_highlighting/test_data/highlighting.html

```diff
@@ -144,14 +144,15 @@
     <span class="variable">y</span><span class="punctuation">;</span>
 
     <span class="keyword">let</span> <span class="keyword">mut</span> <span class="variable declaration mutable">foo</span> <span class="operator">=</span> <span class="struct">Foo</span> <span class="punctuation">{</span> <span class="field">x</span><span class="punctuation">,</span> <span class="field">y</span><span class="punctuation">:</span> <span class="variable mutable">x</span> <span class="punctuation">}</span><span class="punctuation">;</span>
+    <span class="keyword">let</span> <span class="variable declaration">foo2</span> <span class="operator">=</span> <span class="variable mutable">foo</span><span class="punctuation">.</span><span class="unresolved_reference">clone</span><span class="punctuation">(</span><span class="punctuation">)</span><span class="punctuation">;</span>
```

Comment:
`clone` being marked as `unresolved_reference` seems like it would cause the type of `foo2` to not be inferred correctly. Any idea why it doesn't get resolved correctly? `Foo` does derive `Clone`.

Reply:
The `Clone` trait is not defined in the test. Tests don't load the sysroot by default to significantly reduce test time.

Reply:
Ah, yeah. That seems fine then, the type is still inferred correctly because of the method call. Might be worth cleaning up though.